### DISCUSSION TITLE
rest API wrapper. help needed on few points

### DISCUSSION THREAD

**Discussion by 4k5h4y** [Auties00/WhatsappWeb4j#448] [Q&A]
I am creating rest api wrappper over this library. I am using mobile builder to create connection. Can someone please share some code examples on how to request OTP and submit and login once i recieve OTP on sms? I don't know from where to start. Please help me out 😥

**Comment by codespearhead** [comment]
Are you still interested in building a REST layer for this project? Because that's what I'm considering.

By the way, can you answer [this](https://github.com/Auties00/Cobalt/discussions/400#discussioncomment-8780934)?

**Comment by cristiangofiar** [comment]
Hey! Were you finally able to receive the SMS for the OTP connection? or what did you do?
